packages/rum/src/viewCollection.ts

```diff
@@ -55,6 +56,20 @@ export function startViewCollection(location: Location, lifeCycle: LifeCycle, se
     currentView.triggerUpdate()
     currentView.end()
   })
+
+  // Session keep alive
+  const keepAliveInterval = setInterval(
```

Comment:
Unless there is a good reason I'm missing, I'd rather put this logic in the `newView` function where all other `updateView` are done.

Reply:
Since it is global to the viewCollection mechanism and not specific to specific views, what would be the pros/cons for that?

Reply:
We are sending RUM events to keep the session alive. IMO this is not related to the viewCollection mechanism either, but since we chose to send RUM view events for this, we are free to chose where we put them.

By updating the view in the `newView` function:
* all updates are done at the same place: this makes things self contained simpler to follow
* we could imagine a more elaborated logic in the future, to update the view only if hasn't been updated recently (a bit like a debounce)

Reply:
> all updates are done at the same place: this makes things self contained and simpler to follow

I think it depends on what we choose for current `triggerUpdate` method

> we could imagine a more elaborated logic in the future, to update the view only if hasn't been updated recently (a bit like a debounce)

let's see when we get there

One potential drawback of having the keep alive strategy inside the `newView` function is that it would then technically make more sense to start/stop a specific interval for each view which is not really needed for our current problem.

Reply:
Ok